tpot/tpot.py

```diff
@@ -288,6 +289,182 @@ def score(self, training_features, training_classes, testing_features, testing_c
 
         return self._evaluate_individual(self.optimized_pipeline_, training_testing_data)[0]
 
+    def export(self, output_file_name):
+        """Exports the current optimized pipeline as Python code.
+
+        Parameters
+        ----------
+        output_file_name: string
+            String containing the path and file name of the desired output file
+
+        Returns
+        -------
+        None
+        """
+        if self.optimized_pipeline_ == None:
+            raise Exception('A pipeline has not yet been optimized. Please call fit() first.')
+        
+        exported_pipeline = self.optimized_pipeline_
+        
+        # Replace all of the mathematical operators with their results
+        while True:
+            for i in range(len(exported_pipeline) - 1, -1, -1):
+                node = exported_pipeline[i]
+                if type(node) == deap.gp.Primitive and node.name in ['add', 'sub', 'mul']:
+                    val1 = int(exported_pipeline[i + 1].name)
+                    val2 = int(exported_pipeline[i + 2].name)
+                    if node.name == 'add':
+                        new_val = val1 + val2
+                    elif node.name == 'sub':
+                        new_val = val1 - val2
+                    else:
+                        new_val = val1 * val2
+
+                    new_val = deap.gp.Terminal(symbolic=new_val, terminal=new_val, ret=new_val)
+                    exported_pipeline = exported_pipeline[:i] + [new_val] + exported_pipeline[i + 3:]
+                    break
+            else:
+                break
+
+        # Unroll the nested function calls into serial code
+        pipeline_list = []
+        result_num = 1
+        while True:
+            for node_index in range(len(exported_pipeline) - 1, -1, -1):
+                node = exported_pipeline[node_index]
+                if type(node) != deap.gp.Primitive:
```

Comment:
It would support inheritance, but I am not sure if this will be useful/beneficial down the road. A small cosmetic change though:

    if type(node) is not deap.gp.Primitive: 

;)